spec/models/revision_spec.rb

```diff
@@ -28,27 +28,38 @@
 describe Revision, type: :model do
   describe '#references_added' do
     let(:refs_tags_key) { 'feature.wikitext.revision.ref_tags' }
+    let(:wikidata_refs_tags_key) { 'feature.len(<datasource.wikidatawiki.revision.references>)' }
+    let(:wiki) { create(:wiki, project: 'wikidata', language: 'en') }
```

Comment:
This should be named more explicitly, since the tests care about this being a `wikidata` example rather than an arbitrary wiki.